### DISCUSSION TITLE
Allow assignment to nullable

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/csharplang#5801] [General] [answered]
![image](https://user-images.githubusercontent.com/35312807/154870550-119c9b0c-f6a6-4a74-9f71-c54366b3c268.png)
Hey for things that might be null let's implicitly do this
Essentially allows ```MyClassInstance?.MyVariable = "ABC";```

**Comment by chrisoverzero** [comment, marked as answer]
This is a duplicate of the championed #2883, which you are a participant in and (at this time) the last commenter on.

**Reply by Xyncgas** [reply]
huh
